result/neighborsv6.yaml

```diff
@@ -0,0 +1,43 @@
+openapi: 3.1.0
+
+info:
+  title: IPv6 Neighbors models
+  version: ^0.0.0
+
+components:
+  schemas:
+    Neighborsv6.States.Request:
+      description: >-
+        The request to retrieve IPv6 Neighbor state (NDISC cache entries) for device network interface(s).
```

Comment:
The request to retrieve IPv6 Neighbor state (NDISC cache entries) for a network interface(s).